auth/README.md

```diff
@@ -435,20 +464,20 @@ startActivityForResult(
         .build());
 ```
 
-Your application theme could also simply be used, rather than defining a new
-one.
+Your application theme could also simply be used, rather than defining a new one.
 
-If you wish to change the string messages, the existing strings can be
-easily overridden by name in your application. See
-[the built-in strings.xml](src/main/res/values/strings.xml) and simply
-redefine a string to change it, for example:
+If you wish to change the string messages, the existing strings can be overridden
+by name in your application. See the module's [strings.xml](src/main/res/values/strings.xml) file
+and simply redefine a string to change it, for example:
 
 ```xml
 <resources>
     <!-- was "Signing up..." -->
-    <string name="progress_dialog_signing_up">Creating your shiny new account...</string>
+    <string name="fui_progress_dialog_signing_up">Creating your shiny new account...</string>
 </resources>
 ```
+Note: String resource names are internal to the library and there are currently no
+guarantees that they won't change between updates.
```

Comment:
@TheCraftKid haha, yeah. We've broken the strings without remorse several times now, so we want to make sure devs understand that we aren't kidding. 😄